### ISSUE TITLE
Question: Is it possible to point to multiple repo_url's?

### ISSUE THREAD

**Issue by SteveRosam** [backstage/mkdocs-monorepo-plugin#88]
Mkdocs allows you to point to a repo_url, linking the rendered doc directly to the github file.
But with mono repo we have many github repos, is there any way to allow the linking to work?

**Comment by whiskeysierra** [comment]
See #2, #45 and #46.

**Comment by jdoiro3** [comment]
[mkdocs-multirepo-plugin](https://github.com/jdoiro3/mkdocs-multirepo-plugin) lets you import multiple repos with docs into one site and also handles generating the correct edit url links for the pages.

**Comment by srosam** [comment]
> See #2, #45 and #46.

thanks. got it working now. (mostly)

**Comment by srosam** [comment]
> [mkdocs-multirepo-plugin](https://github.com/jdoiro3/mkdocs-multirepo-plugin) lets you import multiple repos with docs into one site and handles generating the correct edit url links for the pages.

yeah I saw that one but had some issue. It wasn't generating the pages for repos it pulled down

**Comment by jdoiro3** [comment]
> > [mkdocs-multirepo-plugin](https://github.com/jdoiro3/mkdocs-multirepo-plugin) lets you import multiple repos with docs into one site and handles generating the correct edit url links for the pages.
> 
> yeah I saw that one but had some issue. It wasn't generating the pages for repos it pulled down

@srosam, feel free to open an issue. I'd like to fix whatever issue you encountered for future users.

**Comment by rafalkrupinski** [comment]
I was luckier with either using symlinks (locally) or checking out (github action) the other repos and pretending it's a monorepo.
